app/src/i18n/locales/en-US.json

```diff
@@ -33,6 +33,13 @@
   "cmps.history.HistoryRowHeader.type": "Type",
   "cmps.history.HistoryRowHeader.created": "Created",
   "cmps.history.HistoryRowHeader.updated": "Updated",
+  "cmps.history.HistoryRow.failure-0": "The swap has failed to complete successfully. Check the logs for additional information.",
```

Comment:
I think `FAILURE_REASON_NONE` (value 0) means "no error", e.g. success. This message won't be displayed in that case, but just for completeness' sake we should add the correct description here.